### PR TITLE
#47 Support inline images

### PR DESCRIPTION
**Pull request by butorov**
Hello
I made support for sending inline images (and add `Content-ID` header to any attachments). See related issue: #47 

If you need any clarifications or code improvements, please, feel free to write me :)

**Related links:**
http://developer.postmarkapp.com/developer-send-api.html ("Inline image attachments")
https://tools.ietf.org/html/rfc2392
https://www.ietf.org/rfc/rfc1806.txt

### CODE REVIEW

**Comment by nicholasserra** [comment]
Thanks for the patch! Will review asap :) Looks like there's some failing tests.

**Comment by butorov** [comment]
Yes, it is :( I will fix test tomorrow.

File: README.md [hunk 1]
```diff
@@ -37,11 +40,6 @@ Version 0.4.6
 Version 0.4.5
 - Fix for Python 3 support (issue #42, PR #43) (mflaxman)
 
-Version 0.4.4
```
Comment (Stranger6667) [inline README.md]: Should it be removed now?

Comment (nicholasserra) [inline README.md]: We're limiting the number of versions shown in the README

**Comment by butorov** [comment]
Hi. I fixed tests (thanks for waiting).
I hope this solution is suitable.
Also, I can squash two last commits.

**Comment by nicholasserra** [comment]
Great, thanks :)

**Pull request merged**